canopen/sdo/client.py

```diff
@@ -561,7 +561,7 @@ def _retransmit(self):
             response = self.sdo_client.read_response()
             res_command, = struct.unpack_from("B", response)
             seqno = res_command & 0x7F
-            if seqno == self._ackseq + 1:
+            if seqno == 1:
```

Comment:
I'm wondering why the `self._ack_block()` call doesn't simply reset the `self._ackseq` attribute to zero in all cases.  Then this check would be fine as is?

Just thinking out loud, let's hear your thoughts on why this is the better place to fix it.